libvcs4j-tools/src/main/java/de/unibremen/informatik/st/libvcs4j/spoon/SpoonModel.java

```diff
@@ -0,0 +1,95 @@
+package de.unibremen.informatik.st.libvcs4j.spoon;
```

Comment:
This class was renamed to [Environment](https://github.com/uni-bremen-agst/libvcs4j/blob/master/libvcs4j-tools/src/main/java/de/unibremen/informatik/st/libvcs4j/spoon/Environment.java). Please remove it.